src/backend/web/routes/feeds.js

```diff
@@ -51,4 +52,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', protect, async (req, res) => {
+  const feedData = req.body;
+  if (!feedData.url && !feedData.author) {
+    res.status(400).json({ message: `URL and Author must be submitted` });
+  }
+  if (await Feed.byUrl(feedData.url)) {
+    res.status(409).json({ message: `Feed for url ${feedData.url} already exists.` });
+    return;
+  }
+  try {
+    const feed = await Feed.create(feedData);
+    res
+      .status(201)
+      .json({ message: `Feed was successfully added.`, id: feed, url: `/feeds/${feed}` });
+  } catch (err) {
```

Comment:
Can you use `error` vs. `err` please, it makes it consistent with the rest of our code.

Reply:
Also fixing the other get routes in the file.